Description: Revert upstream change between 2.30.0 and 2.30.1 to enable a11y by default. This wasn't tested and isn't appropriate for an SRU.
Author: Martin Pitt <martin.pitt@ubuntu.com>

Index: gdm-2.32.0/data/session-setup.entries
===================================================================
--- gdm-2.32.0.orig/data/session-setup.entries	2010-08-27 23:45:29.000000000 +0200
+++ gdm-2.32.0/data/session-setup.entries	2010-12-07 15:23:02.577502001 +0100
@@ -404,13 +404,6 @@
     </entry>
 
     <entry>
-      <key>/desktop/gnome/interface/accessibility</key>
-      <value>
-        <bool>true</bool>
-      </value>
-    </entry>
-
-    <entry>
       <key>/desktop/gnome/accessibility/keyboard/enable</key>
       <value>
         <bool>true</bool>
